reference/7.2/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -451,6 +451,18 @@ intro
 -----
 Once upon a time...
 ```
+Given a list of indices, the operator returns a list of members corresponding to those indices.
+
+```powershell
+-JOIN '123'[ 2 .. 0 ] -EQ '321'
+```
+
+If an object is not an indexed collection, its first element is the object itself.  Index lists are not supported.
+
+```powershell
+(0)[0] -EQ 0
+(0)[0,0] -EQ $NULL
+```
```

Comment:
If an object is not an indexed collection, accessing first element returns the object itself.  Index values beyond the first element return `$null`.

All of those zeros can be confusing. I suggest the following

```powershell
PS> (2)[0] -eq 2
True
PS> (2)[-1] -eq 2
True
PS> (2)[1] -eq $null
True
PS> (2)[0,0] -eq $null
True
```

Style note - all keywords and operators should be lowercase